src/main/java/sggw/wzim/czasnawypad/db/entity/AttractionRating.java

```diff
@@ -0,0 +1,38 @@
+package sggw.wzim.czasnawypad.db.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.Date;
```

Comment:
Don't use `java.util.Date` as it's deprecated. https://www.baeldung.com/java-convert-localdate-sql-date